wiki/Consciousness and Self-Regualtion in Mobile Communication.qmd

```diff
@@ -0,0 +1,18 @@
+---
+title: "J.B. Bayer et al. (2016). Consciousness and Self-Regualtion in Mobile Communication. *Human Communication Research 42 (2016) 71–97."
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: "Judith Kaß"
+categories:
+  - "Digital Media"
+
+**(K):** “By integrating Studies 1 and 2, we see how different forms of media cognition are linked and, at the same time, how they differ from each other in their relationships to overall self-regulation.“ (p. 89)
+**(A):** The paper mostly argues that the media use are a mix of automatic and immersive habits but peoples cognitive and individual tendencies while using media might differ. In this study self-regulation and everybody’s ability to self-regulate is a personal tendency. 
+**(R):** Many studies emphasize the importance of self-regulation and conscious awareness in managing mobile device use and avoiding negative consequences. This article highlights the role of consciousness in shaping individuals' behaviour. Other studies deal with consciousness as present or not present. In this study consciousness is more or less present. This assumption alone allows better insight into behavioural processes. Furthermore it reveals a more complex view on self-regulation and behaviour. It declares impulsiveness as a personal trait that results in better or worse self-regulation toward texting.

```

Comment:
This study shows, that in gaining the ability to self-regulate and gain digital autonomy differs between people. For some it is easy and others struggle with it. With regard to autonomy need satisfaction this study is helpful because understanding that some people struggle more with self-regulation helps designing healthy and empowering media use which can lead to more effective autonomy need satisfaction.